.travis.yml

```diff
@@ -19,7 +19,22 @@ branches:
 env:
   global:
     - TRIAL_REPORTER=text
+    # Version used to run the PYPY tests as defined by pyenv.
+    - PYPY_VERSION=pypy-5.4.1
+    # Some python versions (like PYPY) are installed using pyenv and we cache
+    # the compiled version.
+    - PYENV_ROOT="$HOME/.pyenv"
 
+addons:
+  apt:
+    packages:
+    - libssl-dev
+    - libssl1.0.0
```

Comment:
Obviously this is fine because the build works, but… why don't we need `libffi-dev`?